docs/reference/mapping/fields/source-field.asciidoc

```diff
@@ -6,6 +6,11 @@ at index time. The `_source` field itself is not indexed (and thus is not
 searchable), but it is stored so that it can be returned when executing
 _fetch_ requests, like <<docs-get,get>> or <<search-search,search>>.
 
+ifeval::["{release-state}"=="unreleased"]
+include::synthetic-source.asciidoc[]
```

Comment:
```suggestion
In some cases, particularly where reducing the volume of your stored data is a
priority, you may consider either using <<synthetic-source,synthetic `_source`>>
or <<disable-source-field,disabling the `_source` field>> altogether.

include::synthetic-source.asciidoc[]
```
People may not scroll all the way to the "Disabling the _source field" section so I think it'd helpful to have a short intro sentence here.